curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655b4dad1d38ff7cdd65cbfe.md

```diff
@@ -0,0 +1,1580 @@
+---
+id: 655b4dad1d38ff7cdd65cbfe
+title: Step 74
+challengeType: 0
+dashedName: step-74
+---
+
+# --description--
+
+With everything set in place, call the `pauseSong()`, `setPlayerDisplay()`, `highlightCurrentSong()`, and `setPlayButtonAccessibleText()` functions to correctly update the player.
```

Comment:
Since this is the last step we should add some sort of congrats message like the previous projects did 

```suggestion
With everything set in place, call the `pauseSong()`, `setPlayerDisplay()`, `highlightCurrentSong()`, and `setPlayButtonAccessibleText()` functions to correctly update the player.

Congratulations! Your music player is now complete.
```